Tests/UnitTests/Networking/Backend/BackendGetIntroEligibilityTests.swift

```diff
@@ -74,10 +74,10 @@ class BackendGetIntroEligibilityTests: BaseBackendTests {
         var eligibility: [String: IntroEligibility]?
 
         let products = ["producta", "productb", "productc"]
-        backend.getIntroEligibility(appUserID: Self.userID,
-                                    receiptData: Data.init(1...2),
-                                    productIdentifiers: products,
-                                    completion: {(productEligibility, error) in
+        self.offerings.getIntroEligibility(appUserID: Self.userID,
```

Comment:
Nit: some of these are using `self.offerings` and others `backend.offerings`. Maybe make those consistent?

Reply:
Yeah, missed those that didn't have `self`, I've been updating them as I see em.

Reply:
✅